packages/devtools_app/lib/src/timeline/flutter/timeline_screen.dart

```diff
@@ -0,0 +1,193 @@
+import '../timeline_controller.dart';
+import 'event_details.dart';
+import 'flutter_frames_chart.dart';
+import 'timeline_flame_chart.dart';
+
+class TimelineScreen extends Screen {
+  const TimelineScreen() : super('Timeline');
+
+  @override
+  Widget build(BuildContext context) => const TimelineBody();
+
+  @override
+  Widget buildTab(BuildContext context) {
+    return Tab(
+      text: name,
+      icon: Icon(Octicons.getIconData('pulse')),
+    );
+  }
+}
+
+class TimelineBody extends StatefulWidget {
+  const TimelineBody();
+
+  @override
+  TimelineBodyState createState() => TimelineBodyState();
+}
+
+class TimelineBodyState extends State<TimelineBody> {
+  TimelineMode _timelineMode = TimelineMode.frameBased;
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            _buildLeftAlignedButtons(),
+            _buildRightAlignedButtons(),
+          ],
+        ),
+        if (_timelineMode == TimelineMode.frameBased) FlutterFramesChart(),
+        Expanded(
+          child: Split(
+            axis: Axis.vertical,
+            firstChild: TimelineFlameChart(),
+            secondChild: EventDetails(),
+            initialFirstFraction: 0.8,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildLeftAlignedButtons() {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: [
+        if (_timelineMode == TimelineMode.frameBased) ...[
+          OutlineButton(
+            onPressed: _pauseLiveTimeline,
+            child: const MaterialIconLabel(
+              Icons.pause,
+              'Pause',
+              minIncludeTextWidth: 900,
+            ),
+          ),
+          OutlineButton(
+            onPressed: _resumeLiveTimeline,
+            child: const MaterialIconLabel(
+              Icons.play_arrow,
+              'Resume',
+              minIncludeTextWidth: 900,
+            ),
+          ),
+        ],
+        if (_timelineMode == TimelineMode.full) ...[
+          OutlineButton(
+            onPressed: _startRecording,
+            child: const MaterialIconLabel(
+              Icons.fiber_manual_record,
+              'Record',
+              minIncludeTextWidth: 900,
+            ),
+          ),
+          OutlineButton(
+            onPressed: _stopRecording,
+            child: const MaterialIconLabel(
+              Icons.stop,
+              'Stop',
+              minIncludeTextWidth: 900,
+            ),
+          ),
+        ],
+        const SizedBox(width: 8.0),
+        OutlineButton(
+          onPressed: _clearTimeline,
+          child: const MaterialIconLabel(
+            Icons.block,
+            'Clear',
+            minIncludeTextWidth: 900,
+          ),
+        ),
+        Padding(
+          padding: const EdgeInsets.all(8.0),
```

Comment:
can you make this padding just horizontal to reduce the padding between this list of icons and the top of the app. Bonus points: use the inspector to audit where the extra padding is coming from in this case. Right now, it looks like there is a bit excessive padding between the icons and the tab bar.

Reply:
fixed